### ISSUE TITLE
Action/Button to stop and start all the containers in a group.

### ISSUE THREAD

**Issue by enavarrocu** [RickWong/Captain#16]
<img width="239" alt="image" src="https://github.com/RickWong/Captain/assets/318491/cee7c5e0-6a86-4a7a-9ec8-185ba4052332">

**Comment by RickWong** [comment]
Hey @enavarrocu! This should work already. If you hold Cmd, the group name should say `Start black-magic (0/3)` and if you click it, all containers under it will start.

**Comment by RickWong** [comment]
Btw, was this not discoverable? How could we make this more user discoverable?

**Comment by enavarrocu** [comment]
Indeed, it was not discoverable. At least I didn't think about that. Of course, it makes sense after you know it, but it is not that straightforward. I will think about it.

**Comment by tg44** [comment]
@RickWong  It was not discoverable at all (I learned it from this thread, and I also read the product page, which is not a "normal user behavior" :D ), I think you can add an additional line above the "start captain at login" with the modifier keys state (like cmd, ctrl, shift, option), if I see it I automatically try what it does, and then I see the changes, but from zero, I would never hit modifier keys to see if they do anything.
